packages/core/src/components/pages/FormView/Components/Field.tsx

```diff
@@ -213,6 +215,7 @@ const Field = ({
             required={isRequired}
             readonly={isReadonly}
             nullable={true}
+            popup={isCustom}
```

Comment:
Do not use isCustom.
Go to manageContentOfModel function in forms/studio/display.helpers.ts and add options attribute to enable popup on component when type is date, datetime or time